Cargo.toml

```diff
@@ -7,29 +7,29 @@ license = "GPL-3.0"
 name = "crust"
 readme = "README.md"
 repository = "https://github.com/maidsafe/crust"
-version = "0.30.0"
+version = "0.31.0"
```

Comment:
If the newer Rust compiler version doesn't require us to make any more changes apart from the ones required by Clippy, I'd say it's better to be left at 0.30.0 at this point. Same goes for dependencies if all of them are compatible with Rust 1.24.0 (i.e. if Crust builds fine on the latest stable). Otherwise we might need to check if the updated deps are compatible with the data that is stored on the current Alpha-2 network.